tests/e2e/specs/LicenseDetailsCard.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+
+    'Check if the main div tag with the class selected-license-card is present': function(browser) {
+        browser
+            .init()
+            .assert.elementPresent('.control-label > span')
+            .click('.control-label > span')
+            .assert.elementPresent('.pagination-next')
+            .click('.pagination-next')
+            .assert.elementPresent('.select > select')
+            .click('.select > select')
+            .click('option[value="CC BY-SA 4.0"]')
+            .assert.elementPresent('.selected-license-card')
+    },
+    'Check if the a tag with class license-name redirects to https://creativecommons.org/licenses/by-sa/4.0/?ref=ccchooser': function(browser) {
```

Comment:
This test isn't necessary, it's copied from the LicenseCode tests you submitted, so it should be covered in those tests. LicenseCode would be a better component to handle this test because it's more specific to that component.

Reply:
![licenseCard](https://user-images.githubusercontent.com/43946715/76306163-68e71500-62ec-11ea-8372-67522690c252.jpg)

From the image, this test is for the link highlighted in blue which is a part of `LicenseDetailsCard.vue` component and the link highlighted in yellow is tested in the `LicenseCode.js` and of course, both the links are same

Reply:
Ah I see, nevermind then :)